File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitLog.java
package jp.artan.japanesefoodmod.raw_materials.wood;

import jp.artan.japanesefoodmod.common.wood.FruitLog;
import jp.artan.japanesefoodmod.init.CreativeTabsInit;

public class RawMaterialsFruitLog extends FruitLog {
    public RawMaterialsFruitLog(String name) {
        super(name);
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab);
    }
}

File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitLog.kt
package jp.artan.japanesefoodmod.raw_materials.wood

import jp.artan.japanesefoodmod.common.wood.FruitLog
import jp.artan.japanesefoodmod.init.CreativeTabsInit

class RawMaterialsFruitLog(name: String) : FruitLog(name) {
    init {
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab)
    }
}
